update content for slipped ranking pages (#1444)
pulumi/docs: content/blog/simplify-kubernetes-rbac-in-amazon-eks-with-open-source-pulumi-packages/index.md

production involves creating and managing role-based access control
(RBAC). This is so daunting that RBAC is often not implemented, or
implemented halfway, or the configuration becomes impossible to
maintain. 

Fortunately, Pulumi makes RBAC on Kuberenetes so easy that you'll never create an insecure cluster again. In this post, we will contrast the traditional way of working

> with RBAC on EKS with using Pulumi. Here are a few highlights:

Reviewer: Should we move `Here are a few highlights:` below the `<!--more-->`?

Author: thanks you for checking! Fixed.

configmap, you are only restricted to the resources and verbs allowed in
the namespace "automation" and not in namespace "prod".

## Next Step

In this post, we discussed how setting up Kubernetes RBAC with Pulumi is
simple, comprehensive,
non-sequential and part of your everyday programming experience. You can find the [complete pulumi code for our example](https://gist.github.com/d-nishi/a4e54dfc973ea047ec46c8deb5193f4e) and try it out yourself. 

> Pulumi is open source and free to use. For more examples visit our GitHub examples page

Reviewer 2: ```suggestion
Pulumi is open source and free to use. For more examples, visit our GitHub examples page
```

> [here](https://github.com/pulumi/examples). If you'd like to learn more about Pulumi and how to manage your Kubernetes through code, [click here to get started today] ({{< ref "/docs/quickstart" >}}). 

Reviewer 2: ```suggestion
[here](https://github.com/pulumi/examples). To learn more about Pulumi and how to manage Kubernetes through code, have a look at our ["Get Started with Kubernetes" guide]({{< ref "/docs/quickstart/kubernetes/" >}}). 
```
